python/py-sox/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-sox
+version             1.4.0
+platforms           darwin
+license             BSD
+maintainers         nomaintainer
+
+description         Python wrapper around SoX.
+long_description    ${description}
+
+homepage            https://github.com/rabitt/pysox
+
+checksums           rmd160  e6583059b2e46f28bc76895ccb65f34922fcfbf2 \
+                    sha256  176d3a1e541332a3b581fd60957f684069235fcf61881965f6c9e22e16797a85 \
+                    size    62438
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
```

Comment:
Steve, while this isn't a big deal, should py-sox declare a dependency on sox? (I realize this has already been committed, but perhaps for your next PR...)

Reply:
Thanks for catching this. Fixed in #9858.